crates/loro-internal/src/value.rs

```diff
@@ -406,16 +407,17 @@ impl ApplyDiff for LoroValue {
                         }
                     }
                 } else {
-                    let seq = Arc::make_mut(seq);
-                    for item in diff.iter() {
-                        match item {
-                            Diff::Tree(tree) => {
-                                let mut v = TreeValue(seq);
-                                v.apply_diff(tree);
-                            }
-                            _ => unreachable!(),
-                        }
-                    }
+                    // let seq = Arc::make_mut(seq);
+                    // for item in diff.iter() {
+                    //     match item {
+                    //         Diff::Tree(tree) => {
+                    //             let mut v = TreeValue(seq);
+                    //             v.apply_diff(tree);
+                    //         }
+                    //         _ => unreachable!(),
+                    //     }
+                    // }
+                    unimplemented!()
```

Comment:
Not covered by the current fuzz? How do we ensure the correctness of event now?

Reply:
Use `crates/fuzz` to fuzz it. I have removed the fuzz in `loro-internal`.